grails-app/domain/org/pih/warehouse/order/OrderItem.groovy

```diff
@@ -74,6 +82,8 @@ class OrderItem implements Serializable, Comparable<OrderItem> {
     Date dateCreated
     Date lastUpdated
 
+    Boolean disableRefresh = Boolean.TRUE
```

Comment:
We usually set this to be false by default? Are there too many cases where we want to disable it by default?

Reply:
Yeah, I think it is better to disable it for items and adjustments by default. Otherwise, every flush on related objects will trigger the `afterUpdate` for those and if there will be a lot of items, then a lot of duplicated triggers. So I think for now the answer is yes.